crates/subspace-networking/src/lib.rs

```diff
@@ -30,5 +30,13 @@ mod utils;
 pub use crate::node::{Node, TopicSubscription};
 pub use crate::node_runner::NodeRunner;
 pub use create::{create, Config, CreationError};
+use lazy_static::lazy_static;
 pub use libp2p;
+use libp2p::gossipsub::Sha256Topic;
 pub use pieces_by_range_handler::{PiecesByRangeRequest, PiecesByRangeResponse, PiecesToPlot};
+
+lazy_static! {
+    /// Topic for gossipsub protocol subscription (segment archiving).
+    pub static ref PUB_SUB_ARCHIVING_TOPIC: Sha256Topic =
+        Sha256Topic::new("PUB_SUB_ARCHIVING_TOPIC");
```

Comment:
I think it would be better to use [this](https://doc.rust-lang.org/std/lazy/struct.Lazy.html) from nightly, or just `once_cell` crate. What do you think @nazar-pc

Reply:
I would keep it. Those features are unstable and can be removed, if there is an alternative - it's better to not use it. And [once_cell](https://docs.rs/once_cell/latest/once_cell/) looks like having a different use-case.

Reply:
Hmm, those are literally the same. I meant using this:
https://docs.rs/once_cell/latest/once_cell/sync/struct.Lazy.html